### ISSUE TITLE
Screen Reader: Many Pages: The field of distance has no programmatic name announced

### ISSUE THREAD

**Issue by nbhardwaj-del** [Expensify/App#74868]
## Action Performed
Prerequisites:
- The user is signed in.

1. Using Chrome + JAWS, open the URL - https://new.expensify.com/
2. Navigate to the "+" icon and press Enter
3. Tab to the "Track distance" menu item and press Enter to open the Pop-up
4. Tab to the "Manual" Tab and press Enter to select
5. Tab to the "Distance" field and observe the screen reader announcement

Other Occurrences:
1. On Send Invoice
2. On Create Expense

## Expected Result
When the field of the distance receives focus, the input field should have a programmatic name announced, e.g.:
- 'distance (mi)' or similar

## Actual Result
When the field of the distance receives focus, the screen reader announces the field as “0 edit type in text”; no programmatic name is announced, so that the users cannot identify what information has to be entered.

HTML:
<div dir="auto" class="css-146c3p1" style="color: rgb(175, 187, 176); font-size: 40px; text-align: left; font-family: &quot;Expensify New Kansas&quot;, &quot;Windows Segoe UI Emoji&quot;, &quot;Noto Color Emoji&quot;, &quot;Custom Emoji Font&quot;; font-style: normal; font-weight: 500;">mi</div>

## Area issue was found
Track Distance

## Failed Checkpoint
WCAG 4.1.2 Name, Role, Value
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value

## User Impact
Providing role, state, and value information on all user interface components enables compatibility with assistive technology, such as screen readers, screen magnifiers, and speech recognition software, used by people with disabilities.

## Suggested Resolution
Add a visible name and associate it with the field or use a visually hidden programmatic name, e.g.:
- 'distance (mi)' or similar
https://www.w3.org/WAI/tutorials/forms/labels/

## Environment(s)
Windows Windows 11 Home - Chrome

## Screenshots/Videos
<img width="1919" height="961" alt="Image" src="https://github.com/user-attachments/assets/0ce6198e-6c81-445c-82d8-a3195f2ccec6" />

https://github.com/user-attachments/assets/17d3af8e-cadb-44ec-92ef-cb13424311c0

**Version Number:** NA
**Reproducible in staging?:** NA
**Reproducible in production?:** Yes
**Email or phone of affected tester (no customers):** NA
**Logs:** https://stackoverflow.com/c/expensify/questions/4856 NA
**Notes/Photos/Videos:** Any additional supporting documentation Yes
**Expensify/Expensify Issue URL:** https://platform.applause.com/company/12102/products/36605/community-issues/6997093
**Issue reported by:** NA
**Slack conversation:** NA

**Comment by mallenexpensify** [comment]
`Accessibility` issue, will be addressed soon.

**Comment by mallenexpensify** [comment]
Accessibility, unassigning, will be addressed in bulk once all issues are created.
